views/home.php

```diff
@@ -24,6 +24,7 @@
         <div class="row">
             <div class="col-md-4 order-md-2">
                 <h2>Export to ICS</h2>
+                <p style="color:red" id="warning"></p>

```

Comment:
@StijnCaerts Maybe it's better to use a [Bootstrap alerts](https://getbootstrap.com/docs/4.0/components/alerts/)? Style is your area ;)